### PR TITLE
DP-12883: run npm audit fix to remove vulnerabilities

### PR DESCRIPTION
**Pull request by samseide**
## Description
Update vulnerabilities using npm audit fix

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12883)
- [Github issue](https://github.com/massgov/mayflower/compare/patternlab/DP-12883-update-mf-patternlab?expand=1)

## Steps to Test
1. view the styleguide and make sure nothing looks incorrect while using this branch.

## Screenshots
N/A

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* 

#### @TODO

*

### CODE REVIEW

**Comment by ygannett** [comment]
Looks good.
DP-12883
Once the branch is updated, it's ready to merge.

**Review by ygannett — APPROVED** [review]
LGTM.

File: changelogs/DP-12883.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+___DESCRIPTION___
+Minor
+Added
```
Comment (avrilpearl) [inline changelogs/DP-12883.md]: should be:
```suggestion
Security
```

File: changelogs/DP-12883.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- (Patternlab) DP-12883: Updates patternlab to remove vulnerabilities
```
Comment (avrilpearl) [inline changelogs/DP-12883.md]: can we inc. the packages here i.e.:
```suggestion
- (Patternlab) [Dependency] DP-12883: Bumps browser-sync from 2.26.3 to 2.26.7 and handlebars from 4.05 to 4.1.2 to address security vulnerabilities
```

File: changelogs/DP-12883.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+___DESCRIPTION___
+Minor
+Security
+- (Patternlab) DP-12883: Updates patternlab to remove vulnerabilities
```
Comment (avrilpearl) [inline changelogs/DP-12883.md]:
```suggestion
- (Patternlab) [Dependency] DP-12883: Bumps browser-sync from 2.26.3 to 2.26.7 and handlebars from 4.05 to 4.1.2 to address security vulnerabilities
```

**Review by avrilpearl — APPROVED** [review]

**Pull request merged**